lsst/ts/laser/hardware.py

```diff
@@ -0,0 +1,611 @@
+"""Implements classes corresponding to modules for the TunableLaser.
+
+These classes correspond to one module inside of the TunableLaser. Each class contains child registers that have values
+that can be read and sometimes set. Each class contains a method to publish its registers' values.
+
+Notes
+-----
+These classes are based on the REMOTECONTROL.csv file provided by the vendor.
+
+"""
+import logging
+from lsst.ts.laser.ascii import *
+
```

Comment:
It would be nice to have an explanation of what each of these classes are. I mean, what CPU8000 (and so on) is for or what does it interfaces with?